includes/class-woopay-tracker.php

```diff
@@ -163,7 +194,12 @@ public function should_enable_tracking( $is_admin_event = false ) {
 			return false;
 		}
 
-		// Don't track when woopay is disabled.
+		// Track without checking for .
```

Comment:
This comment looks incomplete.

Reply:
Ooh 🤦 Fixed in https://github.com/Automattic/woocommerce-payments/pull/6870/commits/a6dac21e3015fd9f7ff85ee6c99786a48a7de93b

Thanks Alefe